assets/javascript/app.js

```diff
@@ -164,4 +164,39 @@ $(document).ready(function() {
             $('#show-filter').html('Hide Filter')
         };
     });
-});
\ No newline at end of file
+
+
+
+    //  age verification 
```

Comment:
I would actually suggest using `moment.js` here as you can simplify the code to one line using `.diff` to check if the difference between the current date and the inputed date is > 21 years.

I believe it would be something along the lines of `if (moment().diff(moment(year,month,day), 'y') >= 21) {`